### ISSUE TITLE
Fine tune winning Neural Network

### ISSUE THREAD

**Issue by 14corman** [14corman/JMIProject#19]
This is not a requirement, but I just want to put this here because I think we should try to see just how good we can get the winning model. If, say the best learning rate is between 0.001 and 0.0001, then I will replace all learning rate trials with numbers within that scale such as 0.001, 0.002, 0.004, 0.006, 0.008, 0.0001. This will also happen with drop out rate as well.

**Comment by 14corman** [comment]
Did not need
